.travis.yml

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2020 Arm Limited. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the License); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: sh
+os: linux
+dist: focal
+
+addons:
+  apt:
+    packages:
+      - cmake
+      - srecord
+
+cache:
+  pip: true
+  ccache: true
+  directories:
+    # Cache arm-none-eabi compiler
+    - ${HOME}/.cache/deps
+    # It looks like ccache for arm-none-eabi is not yet supported by Travis.
+    # Therefore manually adding ccache directory to cache
+    - ${HOME}/.ccache
+
+matrix:
+  include:
+
+    # ARM_MUSCA_S1
+
+    - &compile-tests
+      stage: "Compile"
+      name: "Compile Regression and Compliance tests - ARM_MUSCA_S1"
+      env: TARGET_NAME=ARM_MUSCA_S1 CACHE_NAME=ARM_MUSCA_S1
+      language: python
+      python: 3.8
+      install:
+        # Install arm-none-eabi-gcc
+        - pushd /home/travis/build && mkdir arm-gcc && cd arm-gcc
+        - curl -L0 "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2?revision=108bd959-44bd-4619-9c19-26187abf5225&la=en&hash=E788CE92E5DFD64B2A8C246BBA91A249CB8E2D2D" --output gcc-arm-none-eabi-9-2019-q4-major.tar.bz2
```

Comment:
Is there a reason not to use gcc-arm-none-eabi-9-2020-q2-update?

Reply:
The list of supported versions is hardcoded in TF-M 1.1's build system, and 9-2020-q2-update is too new. Also, 9-2019-q4-major is still the version in our documentation: https://os.mbed.com/docs/mbed-os/v6.5/build-tools/index.html#compiler-versions

Reply:
I was asking because 9-2020-q2-update is used in https://github.com/ARMmbed/mbed-os-example-psa/blob/development/.travis.yml/#L45
But 9-2019-q4-major is indeed the official version